Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -124,6 +124,16 @@ extern ASCIILiteral errorAsString(Error);
 #define MESSAGE_CHECK_BASE(assertion, connection) MESSAGE_CHECK_COMPLETION_BASE(assertion, connection, (void)0)
 #define MESSAGE_CHECK_BASE_COROUTINE(assertion, connection) MESSAGE_CHECK_COMPLETION_BASE_COROUTINE(assertion, connection, (void)0)
 
+#define CONDITIONAL_MESSAGE_CHECK_BASE(assertion, connection, completion) do { \
```

Comment:
Is 'completion' used here?

Reply:
Maybe we can be more specific about what is conditional, and call this MESSAGE_CHECK_MAYBE_NULL_BASE or MESSAGE_CHECK_OPTIONAL_CONNECTION_BASE.

Reply:
`completion` is unused here. That was a stray left over.
Okay, let's call it MESSAGE_CHECK_OPTIONAL_CONNECTION_BASE.